src/Hourai/Nitori/Misc.cs

```diff
@@ -197,7 +198,7 @@ public class Misc : HouraiModule {
     }
 
     [Command("lmgtfy")]
-    [Alias("goog", "lmg")]
+    [Alias("goog", "lmg", "google")]
```

Comment:
A "google" alias would imply it actually conducts a google search, which it doesn't. This was to keep with feature parity with RoboNitori, and IMO shouldn't be added.